_posts/2020-04-15-Cleard.md

```diff
@@ -0,0 +1,65 @@
+---
+author: John
+title: Week four co-winner Get Cleard, works to improve global screening processes
+---
+
+![Get Cleard](images/blog/Get Cleard.png)
```

Comment:
This will have to be an absolute URL, you can verify that the image isn't found on our [deployment preview](digitaledmonton.slack.com).

It's easy: just add a forward slash at the start of the URL: `/images/blog/Get Cleard.png`

Reply:
When you said "the image is too big", it confused me because the blog post itself should be resizing it.  I see that you were referring to the blog list page.  It's good practice to specify _where_ you were seeing the error.

<img width="1376" alt="image" src="https://user-images.githubusercontent.com/6334517/79385248-7f087680-7f25-11ea-8fd5-97d728aba32a.png">

I don't have a quick fix (okay I do but I have to finish what I started) and would suggest moving the image just above the first question.  This will cause the blog listing to use your first paragraph as an excerpt:

> This week's competition resulted in a tie...

Reply:
got it...

Reply:
I shouldn't have trusted the "copy path" button on Github clearly.  Tried to avoid miss-typing the path and got burned by copy/paste, and syntax ignorance of course  ; )